src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -21,15 +22,22 @@ class FeatureManager : IFeatureManager
         private readonly ILogger _logger;
         private readonly ConcurrentDictionary<string, IFeatureFilterMetadata> _filterMetadataCache;
         private readonly ConcurrentDictionary<string, ContextualFeatureFilterEvaluator> _contextualFeatureFilterCache;
-
-        public FeatureManager(IFeatureSettingsProvider settingsProvider, IEnumerable<IFeatureFilterMetadata> featureFilters, IEnumerable<ISessionManager> sessionManagers, ILoggerFactory loggerFactory)
+        private readonly FeatureManagementOptions _options;
+
+        public FeatureManager(
+            IFeatureSettingsProvider settingsProvider,
+            IEnumerable<IFeatureFilterMetadata> featureFilters,
+            IEnumerable<ISessionManager> sessionManagers,
+            ILoggerFactory loggerFactory,
+            IOptions<FeatureManagementOptions> options)
         {
             _settingsProvider = settingsProvider;
             _featureFilters = featureFilters ?? throw new ArgumentNullException(nameof(featureFilters));
             _sessionManagers = sessionManagers ?? throw new ArgumentNullException(nameof(sessionManagers));
             _logger = loggerFactory.CreateLogger<FeatureManager>();
             _filterMetadataCache = new ConcurrentDictionary<string, IFeatureFilterMetadata>();
             _contextualFeatureFilterCache = new ConcurrentDictionary<string, ContextualFeatureFilterEvaluator>();
+            _options = options?.Value ?? throw new ArgumentNullException(nameof(options));
```

Comment:
>options?.Value [](start = 23, length = 14)

so we don't expect a default value?

Reply:
The default value should be returned by Value.